soap/namespaces.go

```diff
@@ -36,18 +38,19 @@ const (
 )
 
 // DOM Namespaces
+//
 //nolint:stylecheck
 var (
-	DOM_NS_SOAP_ENV    = dom.Namespace{Prefix: "env", Uri: "http://www.w3.org/2003/05/soap-envelope"}
-	DOM_NS_ADDRESSING  = dom.Namespace{Prefix: "a", Uri: "http://schemas.xmlsoap.org/ws/2004/08/addressing"}
-	DOM_NS_CIMBINDING  = dom.Namespace{Prefix: "b", Uri: "http://schemas.dmtf.org/wbem/wsman/1/cimbinding.xsd"}
-	DOM_NS_ENUM        = dom.Namespace{Prefix: "n", Uri: "http://schemas.xmlsoap.org/ws/2004/09/enumeration"}
-	DOM_NS_TRANSFER    = dom.Namespace{Prefix: "x", Uri: "http://schemas.xmlsoap.org/ws/2004/09/transfer"}
-	DOM_NS_WSMAN_DMTF  = dom.Namespace{Prefix: "w", Uri: "http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd"}
-	DOM_NS_WSMAN_MSFT  = dom.Namespace{Prefix: "p", Uri: "http://schemas.microsoft.com/wbem/wsman/1/wsman.xsd"}
-	DOM_NS_SCHEMA_INST = dom.Namespace{Prefix: "xsi", Uri: "http://www.w3.org/2001/XMLSchema-instance"}
-	DOM_NS_WIN_SHELL   = dom.Namespace{Prefix: "rsp", Uri: "http://schemas.microsoft.com/wbem/wsman/1/windows/shell"}
-	DOM_NS_WSMAN_FAULT = dom.Namespace{Prefix: "f", Uri: "http://schemas.microsoft.com/wbem/wsman/1/wsmanfault"}
+	DOM_NS_SOAP_ENV    = dom.Namespace{Prefix: NSP_SOAP_ENV, Uri: NS_SOAP_ENV}
```

Comment:
I don't think that's any equivalent, because that would make `Prefix` also be `Uri`.

Reply:
Hmm? I don't see that.

![image](https://github.com/masterzen/winrm/assets/224971/a90fc203-ecbd-4f31-96af-c4289d7f9bfc)

Reply:
In [playground](https://go.dev/play/p/7zXVRSo_g1_2)

Reply:
Yes, my bad, I misread the `Prefix` constant as they look very close to each other, sorry :)